packages/http-server/src/__tests__/integration/http-server.integration.ts

```diff
@@ -46,11 +45,7 @@ describe('HttpServer (integration)', () => {
     server = new HttpServer(dummyRequestHandler, serverOptions);
     await server.start();
     await server.stop();
-    await expect(
-      makeRequest({
-        uri: server.url,
-      }),
-    ).to.be.rejectedWith(/ECONNREFUSED/);
+    await expect(httpGetAsync(server.url)).to.be.rejectedWith(/ECONNREFUSED/);
```

Comment:
Does `supertest` hide `ECONNREFUSED` too?

Reply:
IDK. `httpGetAsync` is using Node.js core API, it's the simplest possible solution. That's why I prefer it over `supertest` or `request`.